Source/WebCore/platform/graphics/Damage.h

```diff
@@ -101,6 +136,10 @@ class Damage {
             return;
 
         m_minimumBoundingRectangle.unite(rect);
+        if (m_mode == Mode::BoundingBox) {
+            m_rects[0] = m_minimumBoundingRectangle;
```

Comment:
is it guaranteed `m_rects` will have at least one rect at this point so it won't crash?

Reply:
Yes, because when rectsCount is 0 we always set the bounding box and append it to the vector. This should be covered by the unit tests.

Reply:
I can add an assert.

Reply:
I checked with context and I forgot we now add the rect if `!rectsCount` - it's all good, no need to assert I think